packages/dashboard/package.json

```diff
@@ -13,7 +13,8 @@
     "react": "^16.4.1",
     "react-dom": "^16.4.1",
     "react-grid-layout": "samiyaakhtar/react-grid-layout",
-    "style-loader": "^0.21.0"
+    "style-loader": "^0.21.0",
+    "auto-fontsize": "^1.0.4"
```

Comment:
Who is consuming this?

Reply:
m365. It is in @timtong1982 PR, and I'm helping sleuth why adding the package is causing other things to fail.